src/libs/Middleware/SaveResponseInOnyx.ts

```diff
@@ -10,6 +10,7 @@ const requestsToIgnoreLastUpdateID: string[] = [
     SIDE_EFFECT_REQUEST_COMMANDS.RECONNECT_APP,
     WRITE_COMMANDS.CLOSE_ACCOUNT,
     WRITE_COMMANDS.DELETE_MONEY_REQUEST,
+    WRITE_COMMANDS.RENAME_WORKSPACE_CATEGORY,
```

Comment:
> all these requests are updating the app to the most current state

Is this really the case for `WRITE_COMMANDS.RENAME_WORKSPACE_CATEGORY` request?

Reply:
Yes, I added it to fix the issue of updating the category. After that, when I go back and go to the settings, I see that the category is grayed out even though it was changed
<details>
<summary>VIDEO</summary>

https://github.com/user-attachments/assets/59ce8271-d06d-4e4e-8cd3-bdd4fecf923d

</details>

The RCA: lastUpdateIDFromClient is less than the previousUpdateID received, so we [set](https://github.com/Expensify/App/blob/main/src/libs/Middleware/SaveResponseInOnyx.ts#L39) this response, which did not clear the pendingAction. This issue occurred, and it only happens the first time.

I think this issue is out of the scope of this ticket and has a different RCA. However, if we don't fix it, we will need to perform a regression test. Can you advise?

Reply:
@eVoloshchak Could you review my comment above when you have time? Thank you

Reply:
> The RCA: lastUpdateIDFromClient is less than the previousUpdateID received

@huult, it seems to me that this should be resolved on the backend ideally. Do you agree with that?

Reply:
@eVoloshchak Yes, if they are able to fix it, we should update from the BE side. This issue is not related to this ticket.

Reply:
@eVoloshchak Can we skip this case? It is not related to this issue and has a different RCA. We will report a new ticket to fix it.

FYI: @youssef-lr

Reply:
@eVoloshchak what do you think?

Reply:
> Can we skip this case? It is not related to this issue and has a different RCA. We will report a new ticket to fix it.

Yes, that's exactly what needs to be done. This should be fixed on the back end, it's not related to this PR

Reply:
@eVoloshchak I think we need to confirm with the internal team to avoid regression when testing this case, or we can add this command to the ignore list. Which option do you choose or suggest?

Reply:
@eVoloshchak Could you review this pull request so it can be completed? It has been open for 3 weeks and involves a small change, so your prompt review would be appreciated.

Reply:
@eVoloshchak I have removed this command from the ignore list.